package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "snap-watch-only",
+  "name": "@metamask/snap-watch-only",
   "version": "1.0.0",
```

Comment:
Since this is now just a snap without the dapp I don't see the need for this to be a monorepo, should probably change the configuration of it

Reply:
I agree that this does not need to be a monorepo anymore but this PR is already pretty broad so it would probably be best to change the monorepo configuration in a subsequent pr